bin/uwsgi.ini

```diff
@@ -0,0 +1,8 @@
+[uwsgi]
+module = basket.wsgi:application
+http = :8000
+master = true
+vacuum = true
+enable-threads = true
```

Comment:
does this actually enable threads?

Reply:
because we've had very bad results from enabling threading in Django apps in the past

Reply:
According to the docs:
```
By default the Python plugin does not initialize the GIL. This means your app-generated
threads will not run. If you need threads, remember to enable them with enable-threads.
```

So it is still in worker mode, but it allows Python to run threads in those workers.

Reply:
but we don't want threads in the workers, do we? I just remember we tried running bedrock with several threads and workers and ended up with some very odd behavior, but that was many years ago with Apache mod_wsgi.

Reply:
We can run tests, but my hunch is we want this. There's enough going on with 3rd party libs and threads that I would be worried without it we'd run into trouble. Things like thread locals, and newrelic has thread profiling, and SQLite has thread sharing, etc. This isn't worker threads, from what I understand. It's actually disabling the GIL across the app if this isn't set to `true`.